ansible_collection/hpe/nimble/.github/workflows/sanity_tests.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 6 * * *'
+env:
+  NAMESPACE: hpe
+  COLLECTION_NAME: nimble
+jobs:
+  sanity:
+    name: Sanity (Ⓐ${{ matrix.ansible }})
```

Comment:
What kind of char is that?

Reply:
most likely represents ansible.. have just picked that from ansible documented sanity test shared by ansible reviewer

https://github.com/ansible-collections/collection_template/blob/main/.github/workflows/ansible-test.yml